functions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+function getDB () {
+    $db = new PDO('mysql:host=DB;dbname=collection_app', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+    $plant_query = $db->prepare("SELECT * FROM `plants` LEFT JOIN `plant_types` ON `plants`.`type` = `plant_types`.`id`;");
+    $plant_query->execute();
+
+    $plant_data = $plant_query->fetchAll();
+    if (!empty($plant_data)) {
```

Comment:
error handling checks like this should be done in a seperate function. just grab whatever comes back from the fetchall and return it. then pass that into another function to check for errors